src/v2/controllers/FormController.js

```diff
@@ -32,6 +32,9 @@ export default Controller.extend({
 
   postRender () {
     const currentViewState = this.options.appState.getCurrentViewState();
+    if (!currentViewState) {
```

Comment:
curious when we run into this?
when error happened, we wouldn't render this form, do we?

Reply:
Yeah, it does try to render this form, I think its because BaseLoginRouter calls this.defaultAuth(). This was another reason why I was forcing it to render a terminal view in my previous implementation.

Reply:
maybe remove `this.defaultAuthn`? feel like it makes sense for v1 (render primary auth) but not v2 ?

Reply:
just removing that is not enough, as the BaseController still calls render method which then ends up in postRender within FormController